### PR TITLE
React-week2 homework

### PR DESCRIPTION
**Pull request by Sreelatha-Vallabhaneni**

### CODE REVIEW

**Comment by senner008** [comment]
Hi :)
I see you have put a lot of work into this and I believe in your abilities as a programmer.
But I would like to review this particular app at a later date.
For now I will ask you to make a simple version a todo-app which only has the functionality that is outlined in the homework description. No additional features or css.
This is to make sure that you have a firm understanding of the basic concepts before we start to develop complete apps. 
Hope this makes sense

**Comment by Sreelatha-Vallabhaneni** [comment]
> Hi :)
> I see you have put a lot of work into this and I believe in your abilities as a programmer.
> But I would like to review this particular app at a later date.
> For now I will ask you to make a simple version a todo-app which only has the functionality that is outlined in the homework description. No additional features or css.
> This is to make sure that you have a firm understanding of the basic concepts before we start to develop complete apps.
> Hope this makes sense

Hi Niels, 
I created simple version of todo app as per homework, please review it thank you!

File: react/week2/week2-hw/src/components/timer.js [hunk 1]
```diff
@@ -1,21 +1,17 @@
 import React, { useState, useEffect } from 'react';
 
 function Timer(){
-    //console.log('call function')
     const [count, setCount] = useState(0);
-   // console.log(count)
 
     useEffect(() => {
         setInterval(() => {
```
Comment (senner008) [inline react/week2/week2-hw/src/components/timer.js]: This should be setTimeout instead

File: react/week2/week2-hw/src/App.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import TodoHeader from "./components/todoHeader";
+import Timer from "./components/timer";
+import AddTodo from "./components/todoList";
+
+function App() {
+  return (
+    <div className="App">
+      <TodoHeader/>
+      <Timer/>
+      <AddTodo/>
```
Comment (senner008) [inline react/week2/week2-hw/src/App.js]: Think about what your todo component is doing and name accordingly. Use nouns to describe components. It's doing more than adding a todo. Instead I would prefer Todos or TodoList with Todo child components

File: react/week2/week2-hw/src/components/todoList.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        const newToDos = [...todo];
+        newToDos[index].isChecked = true;
+        setTodo(newToDos);
+    }
+    const removeTodo = (index) =>{
+        const newToDos = [...todo];
+        newToDos.splice(index, 1);
+        setTodo(newToDos);
+    }
+    const RenderList = () => {
```
Comment (senner008) [inline react/week2/week2-hw/src/components/todoList.js]: Same as above. Components should be described using nouns. This component should return a list of Todo components

File: react/week2/week2-hw/src/components/todoList.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+    const submitTodo = (e) => {
+        e.preventDefault();        
+        return setTodo([...todo, {title : "Random Text", id : todo.length + 1}]);        
+    }
+    
+    const handleOnChange = (e) => {
+        return setTodo(e.target.value)
+    }
+    const toggleTodo = (index) => {
```
Comment (senner008) [inline react/week2/week2-hw/src/components/todoList.js]: Please refer to id and not index. This is important when the order is changed and updated. Also, think of these as ids set by the db

**Comment by senner008** [comment]
Good! 👍

**Review by senner008 — COMMENTED** [review]
A few suggestions. Let me know what you think

**Comment by Sreelatha-Vallabhaneni** [comment]
Hi Niels,
I fixed all the issues based on your review and updated it, please verify.
Thanks for your great feedback and support!

**Comment by senner008** [comment]
Ok. 👍

**Pull request merged**